spec.html

```diff
@@ -4264,6 +4264,9 @@ <h1>
             1. [id="step-getvalue-toobject"] Let _baseObj_ be ? ToObject(_V_.[[Base]]).
             1. If IsPrivateReference(_V_) is *true*, then
               1. Return ? PrivateGet(_baseObj_, _V_.[[ReferencedName]]).
+            1. If _V_.[[ReferencedName]] is neither a String nor a Symbol, then
```

Comment:
```suggestion
            1. If IsPropertyKey(_V_.[[ReferencedName]]) is *false*, then
```

Alternatively, I'm not sure why we have that AO instead of just saying "is a property key", since "property key" is a well-defined term. So we could get rid of that AO (as a separate PR) and then just use that phrasing.

Same below `for PutValue`, of course.

Reply:
Sounds good; opened https://github.com/tc39/ecma262/pull/3316.